### PR TITLE
Pronoun - Conditions & death reactions

### PR DESCRIPTION
**Pull request by j-gynn**
## About The Pull Request

Pronoun checks cover `dicts/conditions/*`, `dicts/events/death/death_reactions/*` and `resources/dicts/events/death/mountainous.json`.

Also fixed a potential crash situation with grief strings, where a trait could have strings for the opposite `body` status & attempt to retrieve an empty value.

### CODE REVIEW

File: resources/dicts/conditions/condition_got_strings/gain_congenital_condition_strings.json [hunk 1]
```diff
@@ -1,81 +1,67 @@
...
+        "r_c has been called in by m_c's worried parent. Unfortunately, that worry was well-founded, as m_c seems to have wasting disease.",
+        "m_c seems weaker lately, unable to play as much or explore as far.",
         "m_c is a sickly kit and the Clan fears {PRONOUN/m_c/subject} may not live past kithood. Even if {PRONOUN/m_c/subject} {VERB/m_c/do/does}, this sickness may never leave {PRONOUN/m_c/object}."
     ],
     "blind": [
         "m_c has never had any trouble getting around, but one day {PRONOUN/m_c/subject} {VERB/m_c/ask/asks} a strange question about how someone knew someone else was napping on the other side of camp, and r_c realizes {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} blind.",
-        "m_c fits in perfectly at first, but as {PRONOUN/m_c/subject} {VERB/m_c/grow/grows} and open {PRONOUN/m_c/poss} eyes, it becomes clear that to m_c there's no difference between having {PRONOUN/m_c/poss} eyes closed or open."
-
+        "m_c fits in perfectly at first, but as {PRONOUN/m_c/subject} {VERB/m_c/grow/grows} and open {PRONOUN/m_c/poss} eyes, it becomes clear that to m_c, there's no difference between having {PRONOUN/m_c/poss} eyes closed or open."
```
Comment (scribblecrumb) [inline resources/dicts/conditions/condition_got_strings/gain_congenital_condition_strings.json]:
```suggestion
        "m_c fits in perfectly at first, but as {PRONOUN/m_c/subject} {VERB/m_c/grow/grows} and {VERB/m_c/open/opens} {PRONOUN/m_c/poss} eyes, it becomes clear that to m_c, there's no difference between having {PRONOUN/m_c/poss} eyes closed or open."
```

File: resources/dicts/conditions/healed_and_death_strings/illness_death_strings.json [hunk 2]
```diff
@@ -1,69 +1,68 @@
...
+        "Yellowcough has taken m_c's life."
+    ],
+    "a festering wound": [
+        "m_c has died to a festering wound.",
+        "A festering wound has killed m_c.",
+        "m_c's festering wound has finally killed her."
+    ],
+    "heat stroke": [
+        "m_c died to heat stroke.",
+        "m_c's case of heat stroke has killed her."
```
Comment (scribblecrumb) [inline resources/dicts/conditions/healed_and_death_strings/illness_death_strings.json]:
```suggestion
        "m_c's case of heat stroke has killed {PRONOUN/m_c/object}."
```

File: resources/dicts/conditions/healed_and_death_strings/illness_death_strings.json [hunk 3]
```diff
@@ -1,69 +1,68 @@
...
+    "yellowcough": [
+        "m_c died to yellowcough.",
+        "Yellowcough has killed m_c.",
+        "m_c was killed by yellowcough.",
+        "Yellowcough has taken m_c's life."
+    ],
+    "a festering wound": [
+        "m_c has died to a festering wound.",
+        "A festering wound has killed m_c.",
+        "m_c's festering wound has finally killed her."
```
Comment (scribblecrumb) [inline resources/dicts/conditions/healed_and_death_strings/illness_death_strings.json]:
```suggestion
        "m_c's festering wound has finally killed {PRONOUN/m_c/object}."
```

File: resources/dicts/conditions/healed_and_death_strings/illness_healed_strings.json [hunk 4]
```diff
@@ -78,13 +78,13 @@
...
+        "The stench coming from m_c is finally fading. {PRONOUN/m_c/subject/CAP}{VERB/m_c/'ve/'s} recovered from carrionplace disease.",
         "Against all odds, m_c has recovered from carrionplace disease."
     ],
-	"malnourished": [
+    "malnourished": [
         "m_c finally feels energized again. It seems the newly consistent meals have done {PRONOUN/m_c/object} good."
     ],
-	"starving": [
+    "starving": [
         "m_c's paws no longer feel quite so heavy. {PRONOUN/m_c/subject/CAP} can tell that {PRONOUN/m_c/subject} still need to eat more, but at least the situation isn't as dire.",
```
Comment (scribblecrumb) [inline resources/dicts/conditions/healed_and_death_strings/illness_healed_strings.json]:
```suggestion
        "m_c's paws no longer feel quite so heavy. {PRONOUN/m_c/subject/CAP} can tell that {PRONOUN/m_c/subject} still {VERB/m_c/need/needs} to eat more, but at least the situation isn't as dire.",
```

File: resources/dicts/events/death/mountainous.json [hunk 5]
```diff
@@ -1,40 +1,42 @@
...
-    ]
-  }
+    {
+        "event_id": "gen_death_weasel_death",
+        "location": ["mountainous:camp4"],
+        "season": ["any"],
+        "sub_type": [],
+        "tags": ["clan:kitten"],
+        "weight": 20,
+        "event_text": "m_c sneaks out of camp but finds {PRONOUN/m_c/self} lost in the strange, angular tunnels that lead out into the territory. The Clan search for {PRONOUN/m_c/object} in the morning to no avail. The leftover scent of blood and mountain weasel serves as an intense warning to the other kits.",
```
Comment (scribblecrumb) [inline resources/dicts/events/death/mountainous.json]:
```suggestion
        "event_text": "m_c sneaks out of camp but finds {PRONOUN/m_c/self} lost in the strange, angular tunnels that lead out into the territory. The Clan searches for {PRONOUN/m_c/object} in the morning to no avail. The leftover scent of blood and mountain weasel serves as an intense warning to the other kits.",
```

File: resources/dicts/events/death/death_reactions/parent/parent_dislike.json [hunk 6]
```diff
@@ -1,280 +1,214 @@
...
-    "no_body": [
-
-    ]
-  }
+    "general": {
+        "body": [
+            "As r_c stares down at m_c's body, {PRONOUN/r_c/subject} {VERB/r_c/feel/feels} nothing. m_c was a disappointment.",
+            "r_c sits respectfully, but doesn't contribute a story to m_c's vigil. {PRONOUN/r_c/subject/CAP} {VERB/r_c/know/knows} {PRONOUN/r_c/subject} should feel more about the death of {PRONOUN/r_c/poss} kit, but {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't}.",
+            "It's sad, the vigil. More so because r_c knows {PRONOUN/r_c/subject} should care, but {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't} have any grief for m_c.",
+            "r_c keeps up appearances, soothing the fur of {PRONOUN/r_c/poss} dead kit and sitting vigil, but {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't} care that m_c is dead. {PRONOUN/r_c/subject/CAP} {VERB/r_c/don't/doesn't} know if {PRONOUN/r_c/subject} ever loved {PRONOUN/m_c/object}, not like {PRONOUN/r_c/subject} {VERB/r_c/were/was} supposed to.",
```
Comment (scribblecrumb) [inline resources/dicts/events/death/death_reactions/parent/parent_dislike.json]:
```suggestion
            "r_c keeps up appearances, smoothing the fur of {PRONOUN/r_c/poss} dead kit and sitting vigil, but {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't} care that m_c is dead. {PRONOUN/r_c/subject/CAP} {VERB/r_c/don't/doesn't} know if {PRONOUN/r_c/subject} ever loved {PRONOUN/m_c/object}, not like {PRONOUN/r_c/subject} {VERB/r_c/were/was} supposed to.",
```

File: resources/dicts/events/death/death_reactions/parent/parent_jealousy.json [hunk 7]
```diff
@@ -1,280 +1,214 @@
...
-    "no_body": [
-
-    ]
-  }
+    "general": {
+        "body": [
+            "As r_c stares down at m_c's body, {PRONOUN/r_c/subject} {VERB/r_c/feel/feels} nothing. m_c was a disappointment.",
+            "r_c sits respectfully, but doesn't contribute a story to m_c's vigil. {PRONOUN/r_c/subject/CAP} {VERB/r_c/know/knows} {PRONOUN/r_c/subject} should feel more about the death of {PRONOUN/r_c/poss} kit, but {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't}.",
+            "It's sad, the vigil. More so because r_c knows {PRONOUN/r_c/subject} should care, but {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't} have any grief for m_c.",
+            "r_c keeps up appearances, soothing the fur of {PRONOUN/r_c/poss} dead kit and sitting vigil, but {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't} care that m_c is dead. {PRONOUN/r_c/subject/CAP} {VERB/r_c/don't/doesn't} know if {PRONOUN/r_c/subject} ever loved {PRONOUN/m_c/object}, not like {PRONOUN/r_c/subject} {VERB/r_c/were/was} supposed to.",
```
Comment (scribblecrumb) [inline resources/dicts/events/death/death_reactions/parent/parent_jealousy.json]:
```suggestion
            "r_c keeps up appearances, smoothing the fur of {PRONOUN/r_c/poss} dead kit and sitting vigil, but {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't} care that m_c is dead. {PRONOUN/r_c/subject/CAP} {VERB/r_c/don't/doesn't} know if {PRONOUN/r_c/subject} ever loved {PRONOUN/m_c/object}, not like {PRONOUN/r_c/subject} {VERB/r_c/were/was} supposed to.",
```

**Review by scribblecrumb — CHANGES_REQUESTED** [review]

File: resources/dicts/events/death/mountainous.json (hunk 5 above)
Comment (j-gynn) [reply resources/dicts/events/death/mountainous.json]: Ack this is another British English problem that I wasn't aware was a difference at the time! Though I still maintain y'all are wrong... 😉

**Review by scribblecrumb — APPROVED** [review]

**Pull request merged**